src/components/PassphraseBox.css

```diff
@@ -109,27 +109,26 @@
     cursor: pointer;
     display: block;
     font-size: 1.625rem;
-    transform: scale(1);
-    transition: transform .45s ease;
+    transition: opacity .3s ease;
 }
 
-.passphrase-box .password-skip i.nq-icon {
-    background-image: url('data:image/svg+xml,<svg xmlns="http://www.w3.org/2000/svg" x="0px" y="0px" color="%23ffffff" viewBox="0 0 10 17"><polygon fill="currentColor" points="1.43,17 0,15.58 7.13,8.5 0,1.42 1.43,0 10,8.5 "/></svg>');
+.passphrase-box .password-skip .nq-icon {
+    background-image: url('data:image/svg+xml,<svg xmlns="http://www.w3.org/2000/svg" x="0px" y="0px" color="white" viewBox="0 0 10 17"><polygon fill="currentColor" points="1.43,17 0,15.58 7.13,8.5 0,1.42 1.43,0 10,8.5 "/></svg>');
     margin-left: 0.5rem;
     height: 0.875rem;
     width: 0.875rem;
     transition: transform .3s ease;
 }
 
-.password-skip:hover,
-.password-skip:focus {
-  transform: scale(1.03);
-  outline: none;
+.passphrase-box .password-skip:hover,
+.passphrase-box .password-skip:focus {
+    outline: none;
+    opacity: 0.7;
 }
 
-.password-skip:hover i.nq-icon,
-.password-skip:focus i.nq-icon {
-  transform: translate3D(3px, 0, 0);
+.passphrase-box .password-skip:hover .nq-icon,
+.passphrase-box .password-skip:focus .nq-icon {
+    transform: translate3D(-0.25rem, 0, 0);
```

Comment:
preferred the movement in the other direction.
Also I _might_ prefer to increase the opacity on hover instead the other way around.
But if you prefer it this way, ok.